Makefile

```diff
@@ -60,8 +60,9 @@ test: manifests generate fmt vet ## Run unit tests
 
 ##@ Build
 
+GO=GO111MODULE=on GOFLAGS=-mod=vendor CGO_ENABLED=0 go
```

Comment:
Presumably `CGO_ENABLED=0` is key because the other variables are on by default with the version of Go we currently use, right?

Reply:
That, and -mod=vendor

Reply:
According to https://golang.org/ref/mod#vendoring, "If the `vendor` directory is present in the main module's root directory, it will be used automatically if the `go` version in the main module's `go.mod` file is 1.14 or higher."  So it seems like specifying `GOFLAGS=-mod=vendor` is redundant—but it doesn't hurt anything.

Reply:
Huh, TIL. Still, can't hurt to be explicit.